emoji-ui/src/components/SlackTokenInput.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './SlackTokenInput.css';
+
+class SlackTokenInput extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sourceSlackApiKey: '',
+      destinationSlackApiKey: ''
+    }
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange(e) {
+    const name = e.target.name;
+    this.setState({
+      [name]: e.target.value
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    // saving doesn't actually do anything yet!
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label className="token-input-section">
+          Source Slack Api Key
```

Comment:
💡 Any chance we can add help text to tell them how to get this (the instructions are in the README)

Reply:
Oh yeah, thought of that and then forgot :-)